src/mapping/config/MappingConfiguration.cpp

```diff
@@ -126,7 +126,7 @@ MappingConfiguration::MappingConfiguration(
   }
   {
     XMLTag tag(*this, VALUE_NEAREST_NEIGHBOR_GRADIENT, occ, TAG);
-    tag.setDocumentation("Nearest-neighbour mapping which uses a rstar-spacial index tree to index meshes and run nearest-neighbour queries with a gradient optimization.");
+    tag.setDocumentation("Nearest-neighbor-gradient mapping which uses a rstar-spacial index tree to index meshes and runs nearest-neighbor queries with an linear approximation using gradient data.");
```

Comment:
Sounds like the queries use the linear approximation.
The queries only give the nearest neighbor. Afterwards, for the mapping we use a linear approximation.